css/style.css

```diff
@@ -1,28 +1,43 @@
 * {
     box-sizing: border-box;
+    font-family: 'Hammersmith One', sans-serif;
 }
 h1, h2, h3 {
-    color: darkslategrey;
+    color: white;
+    text-align: center;
+}
+h1{
+  font-size: 4rem;
 }
 header {
     padding: 1rem;
     margin-bottom: 2rem;
     text-align: center;
+    background-color: #086788
 }
 body {
-    background-color: #F3FFDA;
+    background-color: white;
     font-size: 1.5rem;
     padding: 0 50px 0 50px;
     font-family: 'Hammersmith One', sans-serif;
 }
+
+.form-section{
+  background-color: #086788;
+}
+
 form {
     display: flex;
+    justify-content: center;
     box-sizing: border-box;
+    padding: 0 50px 0 50px;
```

Comment:
This line and the line after needs to be deleted. This is taken care of in the body section now.